### PR TITLE
NCC Bennie Bracks, Zoologist

### PR DESCRIPTION
**Pull request by TheBlackMarvel**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Bennie Bracks, Zoologist
+ManaCost:3 W
+Types:Legendary Creature Elf Druid
+PT:3/2
+K:Convoke
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE1 | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of each end step, if you created a token this turn, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt:7]: `Defined$ You` is default for any Defined
`NumCards$ 1` is default for Draw effect

Those params are not doing any harm, and you don't *need* to remove them, but you could if you wanted to.

File: forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Bennie Bracks, Zoologist
+ManaCost:3 W
+Types:Legendary Creature Elf Druid
+PT:3/2
+K:Convoke
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE1 | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of each end step, if you created a token this turn, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1
+SVar:X:Count$ThisTurnEntered_Battlefield_Card.token+YouCtrl
+Oracle:Convoke (Your creatures can help cast this spell. Each creature you tap while casting this spell pays for {1} or one mana of that creature’s color.)\nAt the beginning of each end step, if you created a token this turn, draw a card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt:9]: DeckHints:Ability$Token

File: forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt (hunk 1 above)
Comment (TheBlackMarvel) [reply forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt:7]: Little nervous about this one but I trust you! :D

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt]: You don't have to take my word for it; I wouldn't fault you for testing it 😁

File: forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt (hunk 2 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt]: Looks like you forgot to add the DeckHints

**Review by Northmoc — APPROVED** [review]
👍

**Pull request merged**